source/reference/beta-program/orr.txt

```diff
@@ -13,3 +13,53 @@ Enable Oplog Rollover Resilience
    :class: singlecol
 
 .. include:: /includes/beta-feature.rst
+
+.. include:: /includes/orr-intro.rst
+
+By applying changes earlier in the sync process, ``mongosync`` maintains a more 
+recent position in the :term:`oplog`. This adds resilience to long-running 
+operations, mitigates the risk of oplog rollover, and significantly lowers 
+the risk of restarting the sync.
```

Comment:
(optional suggestion)
Consider putting these benefits into a bulleted list? Not at all mandatory, but it might make it more scannable.

Reply:
Opting to keep this one as is. Though the list would make it more scannable, I feel like the need for a list intro sentence might make it more verbose than necessary!